lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -219,6 +222,41 @@
                 },
 
                 /**
+                 * Render the edit team form.
+                 */
+                editTeam: function (topicID, teamID) {
+                    var self = this;
+                    this.getTeamsView(topicID).done(function (teamsView) {
```

Comment:
This is only being used to get the `teamParams`. Instead we should create a function `get_team_params(topicId)` and call it wherever the context is required.